### PR TITLE
make tags-dropdown overflow in y-direction

### PR DESCRIPTION
**Pull request by BurningTreeC**
the tags-dropdown gets very long with many tags. this PR makes it scrollable

### CODE REVIEW

**Comment by BurningTreeC** [comment]
edit: this pr makes the tag-manager icons-dropdown overflow the same way

File: themes/tiddlywiki/vanilla/base.tid [hunk 1]
```diff
@@ -680,7 +680,7 @@ button.tc-untagged-label {
 	vertical-align: middle;
 }
 
-.tc-edit-tags .tc-block-dropdown {
+.tc-edit-tags .tc-block-dropdown, .tc-tag-manager-table .tc-drop-down {
```
Comment (pmario) [inline themes/tiddlywiki/vanilla/base.tid]: If a CSS rule is valid for several elements, 2 lines should be used like: 

```
.tc-edit-tags .tc-block-dropdown, 
.tc-tag-manager-table .tc-drop-down {
..
}
```
which makes it much more obvious, what's going on.

Comment (BurningTreeC) [reply themes/tiddlywiki/vanilla/base.tid]: thanks @pmario, I've changed it

**Comment by BurningTreeC** [comment]
bump

**Comment by Jermolene** [comment]
Hi @BurningTreeC I try to avoid nested scrollable areas in the core because many browsers don't give a good visual hint that there is out of sight content (and those that do use ugly scroll bars). In this case, one can just scroll the page, which is less fiddly eg on mobile than trying to scroll the subframe.

**Comment by BurningTreeC** [comment]
Ok @Jermolene, that's reasonable

**Pull request closed without merging**